src/systems/systems/src/shared/pid.rs

```diff
@@ -39,6 +40,14 @@ impl PidController {
         self.setpoint = new_setpoint;
     }
 
+    pub fn change_min_output(&mut self, new_min: f64) {
+        self.min_output = new_min;
+    }
+
+    pub fn change_max_output(&mut self, new_max: f64) {
+        self.max_output = new_max;
+    }
```

Comment:
**thought**: Should these perhaps be named `set_...` instead of `change_...` to be in line with common "getter/setter" patterns?

Reply:
Agreed!